content/applications/sales/sales/products_prices/prices/pricing.rst

```diff
@@ -1,146 +1,439 @@
+   :alt: Custom time period pop-up form in Odoo Sales.
+
+From this :guilabel:`Create Period` pop-up form, add a :guilabel:`Name`, :guilabel:`Duration`, then
+a :guilabel:`Unit` (e.g. `Days`, `Weeks`, etc. ). When finished, click :guilabel:`Save and Close`.
+
+Lastly, add the desired price for this time-based rule in the :guilabel:`Price` column.
+
+.. seealso::
+   :doc:`/applications/sales/subscriptions`
+
+Configuration tab
+-----------------
 
-The default pricelist applied to any new customer is *Public Pricelist*. To
-segment your customers, open the customer detail form and change the *Sale
-Pricelist* in the *Sales & Purchases* tab.
+Under the :guilabel:`Configuration` tab, there are a few options that can further customize the
+pricelist.
 
-.. image:: pricing/customer_pricelist.png
+.. image:: pricing/configuration-tab.png
    :align: center
+   :alt: Configuration tab on pricelist detail form in Odoo Sales.
 
-Temporary prices
-----------------
+From here, under the :guilabel:`Availability` section, in the :guilabel:`Country Groups` field,
+certain country groups can be added to the pricelist. There is no limit to how many country groups
+can be added in this field.
 
-Apply deals for bank holidays, etc. Enter start and end dates dates.
+.. note::
+   If no country is set for a customer, Odoo takes the first pricelist without any country group.
+
+Under the :guilabel:`Website` section, there are a few options that can be configured. In the
+:guilabel:`Website` field, this pricelist can be applied to a specific website, if working in a
+multi-website environment. If left blank, the pricelist will be applied to all websites in the
+database.
+
+Click the :guilabel:`Selectable` checkbox to have this pricelist as a selectable option for
+customers to choose as they shop. If the :guilabel:`Selectable` box is left unchecked, customers
+**cannot** select this pricelist for themselves as they shop.
+
+Lastly, there is the option to add an :guilabel:`E-commerce Promotional Code`. To add a code, type
+in the desired promo code that, when entered during the checkout process, applies the pricelist to
+the customer, even if the customer does not fall into the previously-specified criteria.
+
+Show discount percentage to customers
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+With Odoo *Sales*, the option to show the public price *and* the computed discount percentage on the
+product catalog is available.
 
-.. image:: pricing/pricing_period.png
+To do that, navigate to :menuselection:`Sales app --> Configuration --> Settings`, and in the
+:guilabel:`Pricing` section, click the checkbox next to the :guilabel:`Discounts` feature, and then
+click :guilabel:`Save` to save all changes.
+
+After activating the :guilabel:`Discounts` feature, navigate to the :guilabel:`Pricelists` page,
+either by clicking the :guilabel:`Pricelists` link from the :guilabel:`Sales Settings` page, or by
+going to :menuselection:`Sales app --> Products --> Pricelists`.
```

Comment:
Adjusting guis here– 
> navigate to the pricelists page

Since we are not actively on the pricelist page (not showing on the screen), it does not get a guilabel

> from the :guilabel:`Settings` page

The page title says "Settings" and not "Sales Settings" :)

```suggestion
After activating the :guilabel:`Discounts` feature, navigate to the pricelists page, either by
clicking the :guilabel:`Pricelists` link from the :guilabel:`Settings` page, or by going to
:menuselection:`Sales app --> Products --> Pricelists`.
```